src/app/ad-authoring/ad-authoring.state.ts

```diff
@@ -10,7 +10,7 @@ export interface AdAuthoringWorkflowState {
   readonly callToAction?: CallToActionEnum;
   readonly fileSrc?: string;
   readonly file?: File;
-  readonly ampHtml: string;
+  readonly ampHtml?: string;
```

Comment:
Was there a reason to make this optional? It appears a default value is defined and this should always be set for the preview to work right?

Reply:
I made it optional because it my unit tests I am setting the state for landingUrl and file. If ampHtml were required I would have to set ampHtml there as well. Look in download.service.spec.ts line 15.

Reply:
In that test case you can set the ampHtml to be an empty string (or any test value you choose). We should update the test to reflect a valid state instead of changing the requirements of our application state to make test setup easier.